### ISSUE TITLE
Possibility to gather all "ToDos"

### ISSUE THREAD

**Issue by ollehellman** [cmacmackin/ford#162]
Hi,

I'm trying to figure out if there is an easy way to gather all the `@todo something` something to one place/page? It seems my python skills are not up to the task, at least not in a very clumsy way.

The reason I ask, is that it would be really nice to crunch through all the code, see all the fixme things that I have left, gather them in one place to get some overview what needs to be done. Not in a particularly complicated fashion, just something like

## Tasks:

* `todo message` from file `/some/file`
* `another todo ` from file `/another/file`

I can give it a shot if you point me in the correct direction.

**Comment by jacobwilliams** [comment]
I like this idea! It might be useful to be able to specify which `@` fields get sent to their own page. I could see have `@warning` or `@bug` in one place being useful too.

**Comment by hornekyle** [comment]
I agree. Being able to automatically generate lists of todos, bugs and notes along with which file they come from would be a great feature.

**Comment by bilderbuchi** [comment]
FYI, Doxygen does this by default, at least with `@todo`, `@bug` (maybe more).
